packages/4337-local-bundler/src/bin/test.ts

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
```

Comment:
Ported from the original `run.sh` script to JS so that it can be exported as a `bin`.